pkg/collector/corechecks/ebpf/probe/tcp_queue_length.go

```diff
@@ -19,8 +19,8 @@ import (
 	"github.com/iovisor/gobpf/pkg/cpupossible"
 	"golang.org/x/sys/unix"
 
-	bpflib "github.com/DataDog/ebpf"
-	"github.com/DataDog/ebpf/manager"
+	manager "github.com/DataDog/ebpf-manager"
+	bpflib "github.com/cilium/ebpf"
```

Comment:
Take this fwiw..
I would suggest using something like `ebpfmgr` or something for the library alias (and maybe `ciliumebpf` for the other).
We have so many `manager` types/libs/things in the code, that `manager` is quite ambiguous.  If you call it `ebpfmgr` or something, then the `manager.foo` reference becomes much more obvious when it's `ebpfmgr.foo` in the code.

Reply:
FWIW, I tend to agree with this remark.
But I understand that `DataDog/ebpf-manager` has been aliased with `manager` because it’s a drop-in replacement of the previously used `DataDog/ebpf/manager` which wasn’t aliased.

Reply:
true; however, it was ambiguous before, too.  And since we're changing it, we may as well improve that facet as well.